coreclient/src/key_stores/as_credentials.rs

```diff
@@ -13,45 +13,43 @@ use phnxtypes::{
     crypto::signatures::{signable::Verifiable, traits::SignatureVerificationError},
     identifiers::Fqdn,
 };
-use rusqlite::{params, OptionalExtension, ToSql};
+use sqlx::{
+    encode::IsNull, error::BoxDynError, query, query_scalar, Database, Encode, Sqlite,
+    SqliteExecutor, SqlitePool, Type,
+};
 use thiserror::Error;
 use tracing::info;
 
-use crate::{
-    clients::api_clients::ApiClientsError,
-    utils::persistence::{SqliteConnection, Storable},
-};
+use crate::clients::api_clients::ApiClientsError;
 
 use super::*;
 
 pub(crate) enum AsCredentials {
+    // TODO: Why is this unused
```

Comment:
I wonder why this is the case?

Reply:
Should I add a comment here, and we keep it as is, or should I remove this case?